sqlglot/parser.py

```diff
@@ -1917,6 +1920,28 @@ def _parse_convert(self):
             to = None
         return self.expression(exp.Cast, this=this, to=to)
 
+    def _parse_substring(self):
+        this = self._parse_term()
+
+        # Postgres supports the form: substring(string [from int] [for int])
+        # https://www.postgresql.org/docs/9.1/functions-string.html @ Table 9-6
+        matching_delimiter = {
```

Comment:
let's not do it like this, is it possible to instead check if there's a from, and if so just parse the for, or parse the csv?